tests/integration/lambda-authorizer/serverless.yml

```diff
@@ -0,0 +1,29 @@
+service: lambda-authorizer
+
+configValidationMode: error
+
+plugins:
+  - ../../../
+
+provider:
+  httpApi:
+    authorizers:
+      authorizer:
+        type: request
+        functionName: authorizer
+        enableSimpleResponses: true
```

Comment:
I don't see `enableSimpleResponses` anywhere in the current code (being read) nor in your PR. the tests pass _with_ or _without_ this property being set. is that intentional? I must admit I currently don't know much about the topic.

Reply:
Yes, that's right. I'm checking if the response is in simple response format.

Reply:
what I mean is that I don't see any usage of `enableSimpleResponses`, e.g. in a condition anywhere in your code.

also your tests pass as well if I comment out `enableSimpleResponses: true`
```yml
    authorizer:
        type: request
        functionName: authorizer
        # enableSimpleResponses: true
```

Reply:
Yes, I understand your point. I'm not using it because I'm checking if the response is in simple response format without caring enableSimpleResponses flag. That's why if you set to false but your response is in simple format test will pass. I don't think we need to force to use enableSimpleResponses to true, just using a valid format is enough. But if you think is better to check enableSimpleResponses try to do it (no idea how :p but I'll work it out).